content/en/docs/tasks/debug-application-cluster/debug-running-pod.md

```diff
@@ -0,0 +1,136 @@
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Creating a sample application
+
+First, create a sample application to be used with the rest of the steps
+in this task. This example uses a naked pod for simplicity.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sample-app
+spec:
+  shareProcessNamespace: true
+  containers:
+    - name: nginx
+      image: nginx
+```
+
+Create the pod described above in your cluster:
+
+```shell
+kubectl create -f sample-app.yaml
+```
+
+## Create an ephemeral container for debugging
+
+Using `kubectl debug`, create an ephemeral container for debugging:
+
+```shell
+kubectl debug sample-app
+```
+
+The new container now appears in the list of ephemeral containers in a `kubectl
+describe`:
+
+```shell
+kubectl describe pod sample-app
+```
+
+```
+Ephemeral Containers:
+  debugger:
+    Container ID:   docker://208144f2c4bb88e04b3bda7533346c2a606f19d1bd2231d14c236159140ea785
+    Image:          busybox
+    Image ID:       docker-pullable://busybox@sha256:9f1003c480699be56815db0f8146ad2e22efea85129b5b5983d0e0fb52d9ab70
+    Port:           <none>
+    Host Port:      <none>
+    State:          Running
+      Started:      Thu, 29 Aug 2019 13:25:40 +0000
+    Ready:          False
+    Restart Count:  0
+    Environment:    <none>
+    Mounts:         <none>
+```
+
+* You can use arbitrary container image using `kubectl debug --image=$IMAGE_NAME`.
+* Each successive ephemeral container needs a new name. `kubectl debug` defaults
+  to using the name `debugger`. You must name additional containers differently
+  using the `-c` flag to `kubectl debug`.
+
+The busybox container runs `sh` and waits for you to attach using `kubectl`:
+
+```shell
+kubectl attach -it sample-app -c debugger
+```
+
+Once attached you are able to see processes from the other containers because
+of process namespace sharing. For example:
+
+```shell
+% kubectl attach -it sample-app -c debugger
+If you don't see a command prompt, try pressing enter.
+/ # ps auxww
+PID   USER     TIME  COMMAND
+    1 root      0:00 /pause
+    6 root      0:00 nginx: master process nginx -g daemon off;
+   11 101       0:00 nginx: worker process
+   12 root      0:00 sh
+   17 root      0:00 ps auxww
+```
+
+The ephemeral container can be attached and re-attached as long as she shell
+process is running.
+
+Since ephemeral containers allow you to modify the state of the running
+pod, you should recreate the pod so that it's in a known good state. When
+using a {{< glossary_tooltip text="deployment" term_id="deployment" >}} this
+is accomplished by deleting the pod and allowing the
+{{< glossary_tooltip text="replica set" term_id="replica-set" >}} to recreate
+it.
+
+```shell
+kubectl delete pod sample-app
+```
+
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+
+* Learn tips for interacting with processes in other containers in [Share
+  Process Namespace between Containers in a Pod](/docs/tasks/configure-pod-container/share-process-namespace.md)
+
```

Comment:
nit: the link in `whatsnext` is incorrect. Probably a good idea to verify all the links in this PR:
/docs/tasks/configure-pod-container/share-process-namespace/

Reply:
TODO: I'll double check my links once netlify updates for these latest changes.

Reply:
Ok, I've tested all of the links on the netlify preview.